lib/ansible/modules/database/misc/redis.py

```diff
@@ -93,6 +93,14 @@
             - A redis config value.
         required: false
         default: null
+    config_write:
```

Comment:
just bikeshedding but 'persistent' seems a better name.

Reply:
I'm a fan of keeping the similar naming convention that are used in Redis, but then I would call this parameter `config_rewrite` like the Redis command.

I would also suggest adding two additional notes in the description of this parameter:
* This parameter is only valid with the `command: config`
* It will overwrite the whole config, which means it will flush all the current config settings of Redis, not the just the parameter being changed. I know that `CONFIG REWRITE` does that, but it's still good to mention it.